dev/io.openliberty.security.oidcclientcore.internal/resources/io/openliberty/security/oidcclientcore/internal/resources/OidcClientCoreMessages.nlsprops

```diff
@@ -125,3 +125,16 @@ TOKEN_CLAIM_VALUE_MISMATCH.useraction=Ensure the attribute in the OpenID Connect
 TOKEN_MISSING_CLAIMS=CWWKS2425E: The {0} OpenID Connect client token is invalid. The jwtClaims value on the JwtContext object is null or empty.
 TOKEN_MISSING_CLAIMS.explanation=Information is missing from the token and validation of the token cannot be completed.
 TOKEN_MISSING_CLAIMS.useraction=Review the log for any earlier errors for details on the failure. Check the format and content of the token that is returned from the OpenID Connect provider.
+
+TOKEN_EMPTY_CLAIM=CWWKS2426E: The token has an empty [{0}] claim.
+TOKEN_EMPTY_CLAIM.explanation=The claim that is specified in the message must have a valid value.
+TOKEN_EMPTY_CLAIM.useraction=The OpenID Connect provider returns an ID token with an empty claim. When present, the claim must have a valid value. Check the format and content of the token that is returned from the OpenID Connect provider.
```

Comment:
```suggestion
TOKEN_EMPTY_CLAIM.useraction=If the OpenID Connect provider returns an ID token with an empty claim, it must be verified for format. The content must also be checked to ensure that the claim contains a valid value.
```

Reply:
I made the updates per suggestions here except the last sentence in user action. Since this is a suggestion , I will keep it as is..thanks